dotnet/src/Connectors/Connectors.Ollama/Client/OllamaChatRequest.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.SemanticKernel.Connectors.Ollama.Core;
 /// <summary>
 /// Ollama chat request object.
 /// </summary>
-internal sealed class OllamaChatRequest
+public sealed class OllamaChatRequest
 {
```

Comment:
Why did you change this class to public? If only for unit tests, use <internalsvisibleto> instead of making class public.